docs/product/components/toggle-switch.html

```diff
@@ -0,0 +1,249 @@
+            <th class="btw0 bxw0 bbw3 bbw3 bc-red-2 fc-red-500 bg-transparent fs-body1">{% icon "Clear", "mtn1 mbn1" %} Don’t</th>
+            <th class="btw0 bxw0 bbw3 bc-green-2 fc-green-500 bg-transparent fs-body1">{% icon "Checkmark", "mtn1 mbn1" %} Do</th>
+        </thead>
+        <tbody>
+            <tr>
+                <td class="pt24">Toggle switches save and apply changes without an additional step.</td>
+                <td class="p24">
+                    <div class="grid ai-center mb12">
+                        <div class="s-toggle-switch wmx1 mr8">
+                            <input type="radio" name="dont-1-1" id="dont-1-1-on" checked>
+                            <label for="dont-1-1-on">On</label>
+                            <input type="radio" name="dont-1-1" id="dont-1-1-off">
+                            <label for="dont-1-1-off" class="s-toggle-switch--label-off">Off</label>
+                        </div>
+                        <label class="s-label">Feature name</label>
+                    </div>           
+                    <button class="s-btn s-btn__sm s-btn__primary">Save setting</button>
+                </td>
+                <td class="p24">
+                    <div class="grid ai-center mb12">
+                        <div class="s-toggle-switch wmx1 mr8">
+                            <input type="radio" name="do-1-1" id="do-1-1-on" value="On" checked>
+                            <label for="do-1-1-on">On</label>
+                            <input type="radio" name="do-1-1" id="do-1-1-off" value="Off">
+                            <label for="do-1-1-off" class="s-toggle-switch--label-off">Off</label>
+                        </div>
+                        <label class="s-label">Feature name</label>
+                    </div>           
+                </td>
+            </tr>
+            <tr>
+                <td rowspan="2" class="bb bc-black-3 pt24">Toggle switches should only save one thing. If they are part of a larger form with a Save button, sections should be clearly marked.<br><br>Or use <a href="/product/components/radio/">radio buttons</a>.</td>
+                <td rowspan="2" class="bb bc-black-3 p24">
+                    <div class="grid ai-center mb12">
+                        <div class="s-toggle-switch wmx1 mr8">
+                            <input type="radio" name="dont-2-1" id="dont-2-1-on" value="On" checked>
+                            <label for="dont-2-1-on">On</label>
+                            <input type="radio" name="dont-2-1" id="dont-2-1-off" value="Off">
+                            <label for="dont-2-1-off" class="s-toggle-switch--label-off">Off</label>
+                        </div>
+                        <label class="s-label">Feature name</label>
+                    </div>           
+                    <label class="s-label d-block mb4">Feature URL</label>
```

Comment:
Let's get the `for` attributes wired up here. I guarantee you someone will copy and paste from these and grab the markup without it 😋